spark_privacy_preserver/mondrian_utils/utility.py

```diff
@@ -183,23 +183,29 @@ def agg_numerical_column(series):
             else:
                 string = '0-'
         else:
-            min_start = minm[:-1]
-            if(minimum >= int(min_start+'5')):
-                string = min_start+'5-'
+            if (minm[-1]=='0'):
+            string = minm +"-"
```

Comment:
I think you got the wrong indentation level here :)

Reply:
Fixed in 5c2760a